pkg/catalog/fake_graveler_test.go

```diff
@@ -19,7 +19,27 @@ type FakeGraveler struct {
 	hooks                     graveler.HooksHandler
 }
 
-func (g *FakeGraveler) SaveGarbageCollectionCommits(ctx context.Context, repositoryID graveler.RepositoryID, previousRunID string) (runID string, err error) {
+func (g *FakeGraveler) GetRules(ctx context.Context, storageNamespace graveler.StorageNamespace) (*graveler.GarbageCollectionRules, error) {
```

Comment:
why are all those functions added to the FakeGraveler? They were not added to the graveler interface